Translated/npcfunctionname.xml

```diff
@@ -377,10 +377,10 @@
 	<key id="11004843" name="Antiques for Sale" feature="Event" />
 	<key id="11004844" name="이벤트 상인" feature="Event" locale="CN" />
 	<key id="11004845" name="Event Shop" feature="Event" locale="KR" />
-	<key id="11004876" name="로얄 뮤직 아카데미 행정관" feature="EpicApocalypse04" />
+	<key id="11004876" name="Music Academy Head" feature="EpicApocalypse04" />
```

Comment:
Royal Music Academy Administrator?

Reply:
While I _do_ prefer 'Royal Music Academy Administrator', 'Music Academy Head' is the official translation from GMS2